src/Codeception/Module.php

```diff
@@ -345,10 +335,10 @@ protected function getModule($name)
     /**
      * Get config values or specific config item.
      *
-     * @param mixed $key
-     * @return mixed the config item's value or null if it doesn't exist
+     * @param string|null $key
+     * @return array|bool|null the config item's value or null if it doesn't exist
```

Comment:
I think that you missed `string` and `int` in return type and it should be `mixed` too.